src/pages/home/report/ReportActionCompose.js

```diff
@@ -270,10 +229,7 @@ class ReportActionCompose extends React.Component {
             return this.props.translate('reportActionCompose.roomIsArchived');
         }
 
-        if (this.props.report.participants
-            && _.contains(this.props.report.participants, CONST.EMAIL.CONCIERGE)
-            && !_.isEmpty(this.props.blockedFromConcierge)
-            && User.isBlockedFromConcierge(this.props.blockedFromConcierge.expiresAt)) {
+        if (ReportUtils.isBlockedFromConciergeChat(this.props.report, this.props.blockedFromConcierge)) {
```

Comment:
I would recommend that we do the following: 
1. update [this function](https://github.com/Expensify/App/blob/main/src/libs/actions/User.js#L238) 
a. to take the whole `blockedFromConcierge` object
b. if it's empty then return false
c. if it's not empty, then check the `expiresAt` key like it does right now
2. update this code block to be: 

```
if (ReportUtils.isConciergeChatReport(this.props.report) && User.isBlockedFromConcierge(this.props.blockedFromConcierge))
```

Reply:
Okay makes sense. But `ReportUtils.isConciergeChatReport` only looks like 1:1 chats and that `Concierge` exists as a participant.

Where as our check is to check if Concierge exists in any chat. So should I create a another function for this or modify `isConciergeChatReport` to handle group chats as well?
```
const isConciergeChat = report.participants
            && _.contains(report.participants, CONST.EMAIL.CONCIERGE)
```

Reply:
Ah, that's interesting. In that case I would recommend we leave this function as is and create a new one that accommodates DMs. Maybe something like: 

- isConciergeDM
- chatIncludesConcierge

Reply:
I've added a function `chatIncludesConcierge`. I've also updated the function call in `RequestCallPage`. Please check.